src/main/java/redis/clients/jedis/Protocol.java

```diff
@@ -371,4 +371,24 @@ public byte[] getRaw() {
       return raw;
     }
   }
+
+  public static enum EventKeyword implements Rawable {
```

Comment:
Put this in `redis.clients.jedis.args` and the name should contain the word Latency; i.e. LatencyEvent.

Reply:
add `redis.clients.jedis.args.LatencyEvent` instead of EventKeyword